app/lib/services/restApi.ts

```
@@ -1052,58 +1042,60 @@ export const editMessage = async (message: Pick<IMessage, 'id' | 'msg' | 'rid' |
 				appName: getBundleId
 			};
 			try {
 				// RC 0.60.0
-				await sdk.post('push.token', data);
+				await sdk.post('/v1/push.token', data);
 			} catch (error) {
 				console.log(error);
 			}
 		}
 		return resolve();
 	});
```

Biome (2.4.13)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)